src/ngx_http_lua_api.c

```diff
@@ -213,4 +216,74 @@ ngx_http_lua_shared_memory_init(ngx_shm_zone_t *shm_zone, void *data)
     return NGX_OK;
 }
 
+
+ngx_int_t
+ngx_http_lua_delay_list_register(ngx_conf_t *cf, void *tag)
```

Comment:
What exactly is being registered here? Modules requiring delayed init handler invocation?